client/src/components/ReportCollection.js

```diff
@@ -1,8 +1,8 @@
 import { setPagination } from "actions"
 import ButtonToggleGroup from "components/ButtonToggleGroup"
 import {
-  PageDispatchersPropType,
-  mapPageDispatchersToProps
+  mapPageDispatchersToProps,
+  PageDispatchersPropType
 } from "components/Page"
 import ReportCalendar from "components/ReportCalendar"
 import ReportMap from "components/ReportMap"
```

Comment:
This is not directly related to task at hand but I prefer to write here anyway. When there are't any reports with a valid Location with valid coordinates `ReportsMapWidget` doesn't render anything due to `whenUnspecified` prop's default value is null. It is a bit inconvenient in a user point of view since user can display reports in Table, Summary, Calendar, Statistics tabs but nothing is displayed in Map tab. I suggest setting a value for `whenUnspecified` prop in `ReportMap` saying something like `No reports with a Location found.`

Reply:
Did this on another PR: https://github.com/NCI-Agency/anet/pull/3292